tap_teamwork/streams/ticket_search.py

```diff
@@ -1,14 +1,29 @@
-from typing import Dict, Iterator, List
+from typing import Any, Dict, List, Optional
+from datetime import datetime
 from singer import get_logger
-from tap_teamwork.streams.abstracts import FullTableStream
+from tap_teamwork.streams.abstracts import IncrementalStream
 
 LOGGER = get_logger()
 
-
-class TicketSearch(FullTableStream):
+class TicketSearch(IncrementalStream):
     tap_stream_id = "ticket_search"
     key_properties = ["id"]
-    replication_method = "FULL_TABLE"
-    replication_keys: List[str] = []
+    replication_method = "INCREMENTAL"
+    replication_keys: List[str] = ["updatedAt"]
     data_key = "tickets"
     path = "desk/api/v2/search/tickets.json"
+
+    def get_url_params(
+        self, context: Optional[Dict[str, Any]], next_page_token: Optional[Any]
+    ) -> Dict[str, Any]:
+        """Construct URL parameters for the API call including incremental filters."""
+        params = super().get_url_params(context, next_page_token)
+
+        # Add incremental sync filter using updatedAtFrom
+        start_date: Optional[datetime] = self.get_starting_timestamp(context)
+        if start_date is not None:
```

Comment:
More pythonic way, please make this change.

```suggestion
        if start_date:
```

Reply:
This is taken care.